js/components/graph/FocusTab.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FocusModal } from "../common/react_modal.js.jsx";
+
+/**
+ * React component representing an item on the focus menu bar
+ */
+export default class FocusTab extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showFocusModal: false,
+    };
+  }
+
+  /**
+   * Change whether the modal popup describing this focus is shown
+   * @param {bool} value
+   */
+  toggleFocusModal = value => {
+    this.setState({
+      showFocusModal: value
+    });
+  }
+
+  render() {
+    return (
+      <div className={this.props.selected ? "focus active-focus" : "focus"}>
+        <button id={this.props.pId} onClick={() => this.props.highlightFocus(this.props.pId)}>
+          {this.props.focusName}
+        </button>
+        <div className="focus-info">
+          <FocusModal showFocusModal={this.state.showFocusModal} focusId={this.props.pId} onClose={() => this.toggleFocusModal(false)} />
+          {this.props.selected && <button onClick={() => this.toggleFocusModal(true)}>i</button>}
```

Comment:
Add an `aria-label` attribute that acts as a descriptive label for this button (for accessibility).

Reply:
Oh also, I think I wasn't totally clear in my last review. In addition to moving the code in this way, I want you to also move the button in the UI itself so that it is positioned within the tab, rather than in the bottom-right corner.

Reply:
Where inside the tab, though? There is a lot of variation in the length of the text so I am not sure where we should put it. We could put a small icon next to the text, but I want it to be easy to notice as well

Reply:
Right, I'd like to see the button appear to the right of the text, within the tab. I think right now the CSS sets a fixed width for each tab, so for the selected tab you could set a larger width to make room for the button.

Reply:
How does this look?
![image](https://user-images.githubusercontent.com/74102544/128086520-2e797a6e-fdb4-4d18-a97b-d60a5cd4d6e6.png)

Reply:
Probably move the button a bit closer to the left, to create more space from the right border.

Reply:
I pushed my changes! I made it wider when selected. I think this would substitute our original idea of having it's height increase.

Reply:
I edited my earlier comment with a new photo because I did not see that you had already replied, but I did increase the spacing on the right.